packages/api/__tests__/api/private/index.ts

```diff
@@ -1676,6 +1677,7 @@ export type UserFilter = {
 export type UserInput = {
   name: Scalars['String']
   email: Scalars['String']
+  emailVerifiedAt?: Maybe<Scalars['DateTime']>
```

Comment:
Why do we need the maybe type here ?

Reply:
If the email isn't verified yet the property `emailVerifiedAt` will be null. That's why it's maybe.

Reply:
Thanks for your explanations